satpy/etc/areas.yaml

```diff
@@ -433,8 +433,8 @@ met07globe:
   area_extent:
     lower_left_xy: [-5621225.237846375, -5621225.237846375]
     upper_right_xy: [5621225.237846375, 5621225.237846375]
-fci_0deg_1km:
-  description: Full disk FCI view, 1 km (based on pre-launch test data)
+mtg_fci_fdss_1km:
+  description: FCI Full-Disk Scanning Service area definition with 1 km resolution
```

Comment:
Isn't the official full name FDHSI?  Or has this changed?

Should 0deg or some other longitude indicator remain in the name?  What if some FCI full disk will exist elsewhere, in some IODC far future?

Reply:
Thanks for your review! 
According to https://github.com/pytroll/satpy/issues/1248 a consensus (at least for EUM data) was reached for having only the service in the name, as (at least until now - and hopefully in the future) this has always been bound to a location. An area for FCI IODC would be called `mtg_fci_iodc_1km`, as IODC and FDSS are both (full-disk) services. 

As for the FDHSI: FDSS and RSS will be the services names, while FDHSI (Full Disc High Spectral Resolution Imagery) and HRFI (High Spatial Resolution Fast Imagery) are spatial sampling configurations for those services respectively.

Reply:
Ok, thank you for the clarification!